### ISSUE TITLE
Errors in CTSWorldAPIIntegration.cs

### ISSUE THREAD

**Issue by gogman** [adamgoodrich/WorldManager#6]
I am using CTS 1.8.0 and when I add WorldManager v0_10 0901 I get these errors.

_Assets\CTS\Scripts\CTSWorldAPIIntegration.cs(24,40): error CS1061: 'CTSTerrainManager' does not contain a definition for 'RegisterAllShaders' and no accessible extension method 'RegisterAllShaders' accepting a first argument of type 'CTSTerrainManager' could be found (are you missing a using directive or an assembly reference?)_

and 

_Assets\CTS\Scripts\CTSWorldAPIIntegration.cs(24,40): error CS1061: 'CTSTerrainManager' does not contain a definition for 'RegisterAllShaders' and no accessible extension method 'RegisterAllShaders' accepting a first argument of type 'CTSTerrainManager' could be found (are you missing a using directive or an assembly reference?)_

I commented out the offending lines and all seems to work as expected.

**Comment by jjxtra** [comment]
I don't think this is maintained anymore, two years since last commit, nothing on asset store.
